stable/toxiproxy/templates/_helpers.tpl

```diff
@@ -47,6 +47,19 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- end }}
 {{- end -}}
 
+{{- define "toxiproxy_frontend.labels" -}}
+app.kubernetes.io/name: {{ include "toxiproxy.name" . }}-frontend
+helm.sh/chart: {{ include "toxiproxy.chart" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- range $key, $value := .Values.extraLabels }}
+{{ $key }}: {{ $value | quote }}
+{{- end }}
+{{- end -}}
+
```

Comment:
All these labels are applied at the chart level already above. The only difference is you are changing `app.kubernetes.io/name` to `{{ include "toxiproxy.name" . }}-frontend`, which is unconventional (to mix different values in the same chart)

I would just add a `component=frontend` label to frontend deployment, pods, ingress and service labels